geoips/commandline/cmd_instructions.py

```diff
@@ -20,8 +20,8 @@
 import yaml
 
 
-def cmd_instructions_modified(ancillary_dirname):
-    """Check whether or not cmd_instructions.yaml has been modified.
+def instructions_modified(ancillary_dirname, fname="cmd_instructions"):
```

Comment:
See this comment: https://github.com/NRLMMD-GEOIPS/geoips/pull/644/files#r1644854681

Reply:
Fixed